### ISSUE TITLE
How to run ltp-aiodio.part1 on btrfs?

### ISSUE THREAD

**Issue by adam900710** [linux-test-project/ltp#261]
Hi

The test file `runtest/ltp-aiodio.part1` can be executed by `./runltp -f ltp-aiodio.part1` directly.
As it needs the extra junkfiles.

And although test scripts `testscripts/ltp-aiodio.sh` can setup needed files and run that test, it needs extra partition and doesn't support btrfs.

Is there any way to execute `./runltp -f ltp-aiodio.part1` on an existing directory other than a new partition?

**Comment by metan-ucw** [comment]
Yes indeed it misses the junkfile preparation step, we should add mkdir and dd to the runtest file as a preparation steps. You can see the preparation steps we use in openqa here:
https://openqa.opensuse.org/tests/661051/modules/create_junkfile_ltp/steps/1/src
It would make much more sense to move these into the runtest file. I can look into this and prepare a patch for the runtest file later on, or you can try ourself.

The testscript is outdated, we should get rid of it and fix the runtest file instead.

**Comment by kribalac** [comment]
@adam900710 How can I use test script using the command in "ltp-aiodio.part1" so that it does not show 'No such file or directory' messages like mentioned is above comment?
Currently I run as:
 /opt/ltp/runltp -f ltp-aiodio.part1 
and 
/opt/ltp/runltp -f ltp-aiodio.part1  -s aiocp

**Comment by vgs24** [comment]
> @adam900710 How can I use test script using the command in "ltp-aiodio.part1" so that it does not show 'No such file or directory' messages like mentioned is above comment?
> Currently I run as:
> /opt/ltp/runltp -f ltp-aiodio.part1
> and
> /opt/ltp/runltp -f ltp-aiodio.part1 -s aiocp

In the runtest file, you can add mkdir and dd commands.

**Comment by metan-ucw** [comment]
@acerv is currently rewriting these tests, half of them should work without any specific hacks in the latest git and the second half being worked on..

**Comment by bharath1r** [comment]
@metan-ucw and @acerv, we have fix for below issue from ltp-aiodio.part3 module?
/tmp/aiodio.794624/junkfile: No such file or directory

**Comment by metan-ucw** [comment]
@bharath1r the fixes for these runtest files are still ongoing, there is an in-flight patch that is going to be merged soon that fixes ltp-aiodio.part1 I suppose that we will get to the part3 after that.
